bch-devsuite

```diff
@@ -200,6 +200,17 @@ def handle_smartbch():
     elif config.network in (Network.TESTNET, Network.MAINNET):
         config.smartbch = SmartBCHConf(enabled=True)
 
+def handle_bch_rpc_conf():
+    host = inquirer.text(message="BCH RPC host", default="http://bch-node")
+    port = inquirer.text(message="BCH RPC port", default=config.rpc_port())
+    username = inquirer.text(message="BCH RPC username", default="actorforth")
```

Comment:
Can we have it random by default?
We have encountered people messing around to the node with the default password multiple times.
This is a massive complaint I saw in other BCH tools.

Reply:
Updated